pallets/xcm-helper/src/lib.rs

```diff
@@ -168,6 +282,48 @@ impl<T: Config> XcmHelper<BalanceOf<T>, AssetIdOf<T>, T::AccountId> for Pallet<T
         ]))
     }
 
+    fn ump_transact_staking(
```

Comment:
we can merge `ump_transact_staking` and `ump_transact_crowdloan` because staking_currency == relay_currency

Reply:
They are not same. In `ump_transact_staking`, we are subtracting fees from `insurance_pool` after burning fees. 

https://github.com/parallel-finance/parallel/blob/master/pallets/liquid-staking/src/lib.rs#L996-L1001

That's why we put `insurance_pool`  in XCM-helper. 

We can keep this for now and discuss later as a part of #1053

Reply:
@ashi20192020ayush now we use `par/fees` account to manage this, liquid staking just need to add fees inside this account

Reply:
crowdloan pallet is different we will put KSM into that account and burn it

Reply:
@GopherJ This is a logic change. We should do it as a part of separate PR.

cc @ETeissonniere